src/main/java/com/mathworks/ci/MatlabStepExecution.java

```diff
@@ -0,0 +1,101 @@
+package com.mathworks.ci;
+
+/**
+ * Copyright 2019-2020 The MathWorks, Inc.
```

Comment:
I think the copyright starts in the year the file was created (2020). I see this in a few files.

Reply:
Done

Reply:
Looks like this one didn't get pushed?